lms/urls.py

```diff
@@ -206,8 +206,11 @@
         continue
 
     # Make the assumptions that the templates are all in the same dir
-    # and that they all match the name of the key (plus extension)
-    template = "%s.html" % key.lower()
+    # and that they all match the name of the key
+    template = key.lower()
+    if '.' not in template:
+        # Assume html if no file extension specified
+        template = "%s.html" % template
 
     # To allow theme templates to inherit from default templates,
     # prepend a standard prefix
```

Comment:
Probably not in the context of this PR but we should move this theming bit into the theming module.